### PR TITLE
Vision of Event Model added

### PR DESCRIPTION
**Pull request by HristoYonkov**
First example of event model is added.

### CODE REVIEW

**Review by Getsov — DISMISSED** [review]

File: src/models/Event.js [hunk 1]
```diff
@@ -20,32 +20,35 @@ const eventSchema = new Schema({
...
+                minlength: [5, "Hour must be in format: 'hh:mm'!"],
+                maxlength: [5, "Hour must be in format: 'hh:mm'!"],
+            },
+            endTime: {
+                type: String,
+                required: true,
+                minlength: [5, "Hour must be in format: 'hh:mm'!"],
+                maxlength: [5, "Hour must be in format: 'hh:mm'!"],
+            }
+        }, { // Additional date and time (if needed)
```
Comment (Getsov) [inline src/models/Event.js:38]: Is this needed? Isn’t the upper date an array of N date objects?

**Comment by HristoYonkov** [comment]
I've created the Events example Model with all properties. In later stage therse properties will have updated validation!

File: src/models/Event.js (hunk 1 above)
Comment (HristoYonkov) [reply src/models/Event.js:38]: image

Comment (HristoYonkov) [reply src/models/Event.js:38]: I did't work with date objects Much, but if neededd just say it! Instead i put a regexp validation on inputs for models for now :) and has been tested!

File: src/models/Event.js [hunk 2]
```diff
@@ -1,18 +1,38 @@
 const { Schema, model } = require('mongoose');
 
 const eventSchema = new Schema({
-    eventName: { type: String, require: true, },
+    title: {
+        type: String,
+        require: true,
```
[the remark this answers is not in the document]
Comment (HristoYonkov) [reply src/models/Event.js]: Like i said, it is a typo :) But thanks! However, mongoose did not warn me.. wich is strange!

**Review by Getsov — APPROVED** [review]

**Pull request merged**